common/utils/StringUtils.cpp

```diff
@@ -109,22 +108,9 @@ bool StripSuffix(string *s, const string &suffix) {
   }
 }
 
-string IntToString(int i) {
-  ostringstream str;
-  str << i;
-  return str.str();
-}
-
-string IntToString(unsigned int i) {
-  ostringstream str;
-  str << i;
-  return str.str();
-}
-
 string IntToHexString(unsigned int i, unsigned int width) {
```

Comment:
IntToHexString isn't used in our codebase now. I don't plan on porting this to the ola::strings